src/esp/metadata/attributes/ObjectAttributes.h

```diff
@@ -406,11 +402,11 @@ class StageAttributes : public AbstractObjectAttributes {
   std::string getAbstractObjectInfoInternal() const override {
     std::string res = getNavmeshAssetHandle();
     res.append(1, ',')
-        .append(cfg.value("gravity"))
+        .append(getAsString("gravity"))
         .append(1, ',')
-        .append(cfg.value("origin"))
+        .append(getAsString("origin"))
         .append(1, ',')
-        .append(cfg.value("light_setup"));
+        .append(getLightSetup());
     return res;
   }
```

Comment:
Here too.

Reply:
I'm going to strip out all of the offending functionality from this PR and refactor all of these to use the formatString process in another PR.